spreadsheet_oca/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "name": "Spreadsheet Oca",
     "summary": """
         Allow to edit spreadsheets""",
-    "version": "16.0.1.2.3",
+    "version": "16.0.1.2.4",
```

Comment:
this change is not required. It is done when merging the module, calling ocabot merge action.

Reply:
Ah I missed that, sorry. Now corrected and force pushed.